### DISCUSSION TITLE
Image Message cache?

### DISCUSSION THREAD

**Discussion by theowenyoung** [flyerhq/flutter_chat_ui#167] [Ideas & Requests] [answered]
Can we cache the image for the image message? or maybe add an argument `Function fetchImage(String url)`?

**Comment by demchenkoalex** [comment]
There is `imageMessageBuilder` I guess you can build it using some caching image library?

**Reply by theowenyoung** [reply]
Thanks! I think that's what I need! Thanks again for the great project!

**Comment by demchenkoalex** [comment, marked as answer]
added natively in v2 https://github.com/flyerhq/flutter_chat_ui/issues/646
